torch/_inductor/compile_worker/subproc_pool.py

```diff
@@ -306,7 +298,6 @@ def _warm_process_pool(pool: AnyPool, n: int) -> None:
 
     # We force them to start here with some YOLOing of the internal methods.
 
-    # TODO(masnesral): Are these still relevant?
```

Comment:
Turns out this does speed up initialization of the ProcessPoolExecutor we use _inside_ the SubprocPool